how-to/virtualisation/libvirt.md

```diff
@@ -251,7 +251,29 @@ RSI=0000000000000000 RDI=ffff8f0fdd5c7e48 RBP=ffff8f0f5d5c7e18 RSP=ffff8f0f5d5c7
 
 ## Huge pages
 
-Using huge pages can help to reduce TLB pressure, page table overhead, and speed up some further memory relate actions. Furthermore by default [transparent huge pages](https://www.kernel.org/doc/Documentation/vm/transhuge.txt) are useful, but can be quite some overhead - so if it is clear that using huge pages is preferred then making them explicit usually has some gains.
+Let's start with a summary of the allocation and structuring of memory in an OS, and its relationship to transparent huge pages and {term}`huge pages <hugepage>`.
+
+When you launch an application, the OS allocates virtual memory to it as a range of virtual addresses. The virtual memory is fake; it only allows the application to think it has more memory than what’s physically available.
+
+However, the CPU architecture will determine the amount of virtual memory allocated to the application; [64-bit CPUs](https://en.wikipedia.org/wiki/64-bit_computing) require 16 EB, whereas [32-bit CPUs](https://en.wikipedia.org/wiki/32-bit_computing) require 4 GB.
```

Comment:
Might it be clearer to say "support" rather than "require" for these two cases?  Require might make it sound like applications have to have that amount allocated, whereas in reality it just means that is the addressable space available.